README.md

```diff
@@ -1281,6 +1333,95 @@ const { data, to } = await sdk.vault.setDepositDataManager.encode(params)
 const gas = await sdk.vault.setDepositDataManager.estimateGas(params)
 ```
 ---
+### `sdk.vault.createEigenPod`
+
+#### Description:
+
+Adding eigen pod to the vault. Only for restake vaults and only restake operators manager can perform this action.
+
+#### Arguments:
+
+| Name           | Type     | Required | Description |
+|----------------|----------|----------|-------------|
+| userAddress    | `string` | **Yes**  | - |
+| vaultAddress   | `string` | **Yes**  | - |
+
+#### Example:
+
+```ts
+const params = {
+  vaultAddress: '0x...',
+  userAddress: '0x...',
+}
+
+// Send transaction
+const hash = await sdk.vault.createEigenPod(params)
+// When you sign transactions on the backend (for custodians)
+const { data, to } = await sdk.vault.createEigenPod.encode(params)
+// Get an approximate gas per transaction
+const gas = await sdk.vault.createEigenPod.estimateGas(params)
+```
+---
+### `sdk.vault.setEigenPodOperator`
+
+#### Description:
+
+Adding operator to the current eigen pod. Only for restake vaults and only restake operators manager can perform this action.
```

Comment:
Mb something like that?
This action is specific to restake vaults and can only be executed by the restake operators manager.

Reply:
https://github.com/stakewise/v3-sdk/pull/122/commits/210ca5e6df93a0afb07491dae920bdce608a2a9b#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1369